python/tskit/trees.py

```diff
@@ -3860,6 +3860,160 @@ class TreeSequence:
     the :meth:`.variants` method iterates over all sites and their genotypes.
     """
 
+    individuals_flags: np.array
+    """
+    Efficient access to the ``flags`` column in the
+    :ref:`sec_individual_table_definition` as a numpy array (dtype=np.uint32).
+    Equivalent to ``ts.tables.individuals.flags``.
```

Comment:
"Equivalent to" makes it sound like there's no difference (the "Efficient" above notwithstanding). Maybe "Obtains the same values as `xxx` but without copying the entire tables."? Or, maybe it's okay?

Reply:
I've added an extrac note about this